src/config/consts.js

```diff
@@ -77,6 +77,7 @@ export const consts = {
   URL_REGEX:
     /^https?:\/\/(?:www\.)?[-a-z0-9@:%._\+~#=]{1,256}\.[a-z0-9()]{1,6}\b(?:[-a-z0-9()@:%_\+.~#?&\/=]*)$/i,
   VIDEO_TYPE_REGEX: /\.(swf|avi|flv|mpg|rm|mov|wav|asf|3gp|mkv|rmvb|mp4)$/i,
+  UMLAUT_REGEX: /[\u00C0-\u017F]/g,
```

Comment:
does this be exact the same as `/[üäöÜÄÖß]/g` or match more?

Reply:
Yes, practically the same.

Reply:
i think my suggestion is more readable, would you like to change it?

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/630f877b1989fdb95f3832ea9b40ded03862c507